File: app/src/main/java/com/example/meyss/javaretrofitdagger/data/AppDatabase.java
package com.example.meyss.javaretrofitdagger.data;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {Pokemon.class,Attack.class,},version =2,exportSchema = false)
public abstract class AppDatabase extends RoomDatabase{

    public abstract PokemonDAO pokemonDao();
}

File: app/src/main/java/com/example/meyss/javaretrofitdagger/data/AppDatabase.kt
package com.example.meyss.javaretrofitdagger.data

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [Pokemon::class, Attack::class], version = 3, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {

    abstract fun pokemonDao(): PokemonDAO
}
